DXClient.sln

```diff
@@ -128,19 +128,13 @@ Global
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresGLRelease|x86.ActiveCfg = AresGLRelease|x86
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresGLRelease|x86.Build.0 = AresGLRelease|x86
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNADebug|Any CPU.ActiveCfg = AresXNADebug|Any CPU
-		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNADebug|Any CPU.Build.0 = AresXNADebug|Any CPU
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNADebug|ARM64.ActiveCfg = AresXNADebug|ARM64
-		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNADebug|ARM64.Build.0 = AresXNADebug|ARM64
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNADebug|x64.ActiveCfg = AresXNADebug|x64
-		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNADebug|x64.Build.0 = AresXNADebug|x64
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNADebug|x86.ActiveCfg = AresXNADebug|x86
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNADebug|x86.Build.0 = AresXNADebug|x86
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNARelease|Any CPU.ActiveCfg = AresXNARelease|Any CPU
-		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNARelease|Any CPU.Build.0 = AresXNARelease|Any CPU
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNARelease|ARM64.ActiveCfg = AresXNARelease|ARM64
-		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNARelease|ARM64.Build.0 = AresXNARelease|ARM64
 		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNARelease|x64.ActiveCfg = AresXNARelease|x64
-		{2CD114E3-C30F-4A95-9B4A-455C2C8F82E5}.AresXNARelease|x64.Build.0 = AresXNARelease|x64
```

Comment:
@frg2089 why exactly "disable build XNA when current platform are not x86"? I can understand ARM64, though isn't there some sort of x86 emulation?

Reply:
Ah so it's just that the non-x86 builds are disabled.